corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -475,6 +477,35 @@ Formplayer.ViewModels.EvaluateXPath = function() {
         };
         $.publish('formplayer.' + Formplayer.Const.EVALUATE_XPATH, [self.xpath(), callback]);
     };
+
+    self.matcher = function(flag, subtext) {
+        var match, regexp;
+        // Match text that starts with the flag and then looks like a path.
+        // CKEditor reserves the right to insert arbitrary zero-width spaces, so watch for those.
```

Comment:
ripped shamelessly from vellum. i figured if it was relevant for vellum it'd be relevant for this too. i figured it was some sort of copy pasting thing?